### DISCUSSION TITLE
Details List control - Getting Error loading control error

### DISCUSSION THREAD

**Discussion by ramsathiamoorthy** [microsoft/powercat-creator-kit#136] [Q&A] [answered]
I am trying to show the data from a Sharepoint list with a Fluent Details list control, I created a collection and populated it on start and assigned the table column properties as stated in the example, but i am always getting error loading control no matter what ever sharepoint list i change to. Any idea how to debug this error

**Comment by denise-msft** [comment]
@ramsathiamoorthy Can you please read the basic usage section in the documentation and let me know if that helps? There are some specific steps needed to render items in DetailsList properly: https://learn.microsoft.com/en-us/power-platform/guidance/creator-kit/detailslist#basic-usage

**Comment by yaseralhosani** [comment, marked as answer]
Hi @ramsathiamoorthy,

I know this is an old post, but I would share a solution based on my practice in case anyone faces the same issue.

The DetailsList Sometimes shows an error loading when playing around with add/remove items and add/remove fields.
Just cut and paste it and it will solve this issue. If not, check with the data source/collection and remove unused fields.
